ui/app/templates/components/calendar-widget.hbs

```diff
@@ -11,7 +11,7 @@
   </D.Trigger>
   <D.Content class="popup-menu-content calendar-content">
     <nav class="box menu">
-      <div class="calendar-title popup-menu-item">Date options</div>
+      <div class="calendar-title is-subtitle-gray">DATE OPTIONS</div>
```

Comment:
You could instead do this with the `.calendar-title` styling with `text-transform:uppercase`

Reply:
I removed that text-transform property from calendar-title actually, since we use it a few different places in Ivana's designs, so I pulled it out into it's own class